python_modules/dagster-test/dagster_test/graph_job_op_toys/run_status_sensors.py

```diff
@@ -42,7 +45,7 @@ def status_job():
 @run_status_sensor(pipeline_run_status=DagsterRunStatus.SUCCESS, request_job=status_job)
 def succeeds_sensor(context):
     if context.dagster_run.pipeline_name != status_job.name:
-        return RunRequest(
+        yield RunRequest(
```

Comment:
ah... I think we should add a comment here to the effect of:

`keeping this as yield even though we encourage return values so that we test the thingggg`

And maybe split this out as a separate sensor called `yields_run_request_sensor`